stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -549,7 +550,7 @@ internal class ChatDomainImpl internal constructor(
         activeChannelMapImpl.values.toList()
 
     fun generateMessageId(): String {
-        return currentUser.id + "-" + UUID.randomUUID().toString()
+        return (_user.value?.id ?: "user_not_user") + "-" + UUID.randomUUID().toString()
```

Comment:
Do we really want to silently swallow the error here? Not having a user for this function is an unexpected invalid state, isn't it?

Reply:
Changed the signature for the method, now it is a pure function. No need to use `_user.value` anymore.